doc/nrf/protocols/matter/getting_started/tools.rst

```diff
@@ -221,12 +221,47 @@ ZAP tool west commands
 
 To simplify work with the :ref:`ug_matter_gs_tools_zap`, there are two commands implemented within the Matter west commands set:
 
+* ``zap-append``
 * ``zap-gui``
 * ``zap-generate``
 
 By default, both commands look for a :file:`.zap` file in the current directory.
 You can also specify the path to the :file:`.zap` file explicitly.
 
+.. _ug_matter_gs_tools_matter_west_commands_append:
+
+``zap-append`` command
+------------------------------
+
+To simplify adding new custom clusters to the Matter ZCL database and generating the required C++ files, you can use the ``zap-append`` west command.
+The command converts the provided XML files with the cluster definition to the JSON format and adds them to the data model definition file (:file:`zcl.json`).
+Using this command, you can add the cluster to the existing data model definition file located in :file:`<default Matter SDK location>/src/app/zap-templates/zcl/zcl.json` or create a new one.
```

Comment:
> or create a new one.

A new file, right? Sorry, but from the context it's a bit unclear to me if we create new cluster or new json file.

Reply:
Changed and explained it better.